ai-gateway/concepts/providers.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Providers
+description: Understanding AI providers in the AI Gateway
+---
+
+Providers are the AI services that host and serve models. The AI Gateway routes your requests to providers and handles authentication, failover, and observability.
+
+## Built-in providers
+
+The AI Gateway includes built-in support for these providers:
+
+| Provider | Description |
+|----------|-------------|
+| **OpenAI** | GPT models, o1/o3 reasoning models |
+| **Anthropic** | Claude models |
+| **Google** | Gemini models |
+| **DeepSeek** | DeepSeek Chat and Reasoner models |
+| **OpenRouter** | Multi-provider routing service |
+| **Hyperbolic** | Open-source model hosting |
+| **InceptionLabs** | Diffusion-based language models |
+| **Inference.net** | Distributed inference network |
+
+For the complete list of models available from each provider, see the [Model Catalog](/ai-gateway/reference/model-catalog).
+
+## Operating modes
+
+### Passthrough mode (default)
+
+When no providers are configured, the gateway operates in **passthrough mode**:
+
+- All built-in providers are available
+- Your SDK's API key is forwarded directly to the provider
+- Automatic retries and metrics are still enabled
```

Comment:
no retries if we don't have a different model/key to try. right now do never retry a key/model combo.